modules/ocf_mirrors/files/record-mirrors-stats.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
```

Comment:
python files should either drop the `.py` if they're not intended to be imported (and only executed), or use underscores so they can be imported

our tradition is typically to drop the extension